src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationPredictedVsObservedChartPresenter.cs

```diff
@@ -77,7 +77,7 @@ protected override void AddRunResultToChart(ParameterIdentificationRunResult run
          addPredictedVsObservedToChart(runResult.BestResult.SimulationResults, (column, curve) =>
          {
             curve.Description = CurveDescription(curve.Name, runResult);
-            curve.Name = column.PathAsString;
```

Comment:
Behavior was always to just use the `PathAsString` since PI but we have this chart method called `NameForColumn` available